packages/react/src/primitives/Alert/Alert.tsx

```diff
@@ -34,10 +34,16 @@ const AlertPrimitive: Primitive<AlertProps, typeof Flex> = (
     }
   }, [setDismissed, onDismiss, dismissed]);
 
+  variation = variation || 'default';
```

Comment:
yeah this was the approach I was going with at first but reverted back because I wanted the `data-variation/size` attributes to also be updated.  I was thinking that it would be better for a customer to continue to use the data attribute selectors to update the styling (that way they don't have to worry about import ordering because their rules will all be more specific than ours).

Reply:
Customers would still be able to use data attributes, but just 'default' ones would not have a data attribute (which is the case today). We initially didn't include 'default' data attributes to keep the HTML tidy and so customers don't need to add `[data-variation='default']` to target the default variation. Whether or not we have the data attributes is less important, but typically BEM doesn't include 'default' modifiers

Reply:
currently the overrides to defaults don't affect the variations because the variation selectors have a greater specificity than the default state, with this flattened specificity don't we need a way to target the default state so that we can update it without overriding the other variations?